src/test/regress/multi_schedule

```diff
@@ -17,6 +17,7 @@
 # ---
 test: multi_extension
 test: multi_table_ddl
+test: multi_schema_support
```

Comment:
You really like having multi_schema_support here :)

Now we have two copies of multi_schema_support!